eclair-core/src/main/scala/fr/acinq/eclair/payment/PaymentPacket.scala

```diff
@@ -42,16 +42,16 @@ object IncomingPaymentPacket {
 
   // @formatter:off
   /** We are the final recipient. */
-  case class FinalPacket(add: UpdateAddHtlc, payload: PaymentOnion.FinalPayload) extends IncomingPaymentPacket
+  case class FinalPacket(add: UpdateAddHtlc, payload: PaymentOnion.FinalTlvPayload) extends IncomingPaymentPacket
```

Comment:
This is a temporary change, right? When we add support for receiving blinded payments, we'll need `FinalPacket` to work for both blinded and non-blinded payloads somehow.

Reply:
Yes.